deep-learning/_posts/2018-01-17-SBNet.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: review
+title:  "SB-Net: Sparse Block Network for Fast Inference"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/abs/1801.02108   
+cite:
+  authors: "Mengye Ren, Andrei Pokrovsky, Bin Yang, Raquel Urtasun"
+  title:   "SB-Net: Sparse Block Network for Fast Inference"
+  venue:   "--"
+---
+
+### Description
+The authors proposed a novel non-sparse convolution algorithm. They use a mask that is known and can apply to input images.for example, in a autonomous driving, only the areas on the roads are interested and be accessible from the off-line mode. This mask is a binary(active/inactive) to represent the available regions for CNN, and all operation will be placed on the active part of images.The binary mask has been converted to a list of indices, where each index belong to the location of the corresponding pixel (Fig:1 ).
```

Comment:
images.  For

Reply:
what is the off line mode?

Reply:
This mask is binary(active/inactive) and represents the  regions available for the CNN to process (?)

Reply:
all operations

Reply:
of the images

Reply:
belongs

Reply:
How do you get to know where the road is?  Maybe add a sentence on that.